### ISSUE TITLE
Clarify need for --security-opt in docs

### ISSUE THREAD

**Issue by jareware** [femtopixel/docker-google-lighthouse#12]
Some environments (e.g. Bitbucket Pipelines) [don't allow options such as `--security-opts` for Docker](https://confluence.atlassian.com/bitbucket/run-docker-commands-in-bitbucket-pipelines-879254331.html), but as it turns out, this image runs just fine without that option. 🎉 

So... why is this option suggested in the docs? Am I missing something? My reports (seem to) contain all the expected things.

**Comment by jaymoulin** [comment]
The option is suggested to avoid use of `--cap-add=SYS_ADMIN` which is a bit overkill (option you may use in your pipeline).

As `--security-opt` and `--cap-add` are Docker specific parameters, I won't explain in details how to use it. A better documentation already cover this topic : https://docs.docker.com/engine/security/seccomp/

I'm glad you were able to make report through your pipeline (which is the targeted purpose of this image ;))

**Comment by jareware** [comment]
Thanks for the thorough response!

I may have worded my question poorly. Let me try again:

Should the docs continue to suggest the use of `--security-opt` and `--cap-add` **even though it seems they're not needed**? I'm getting the exact same Lighthouse report with and without those options.

Feel free to ignore me if you feel like this is irrelevant, just wanted to try and help others who might want to run this image in an environment that doesn't allow using those options. :)

**Comment by jaymoulin** [comment]
Oh! you're right these parameters seems not needed anymore with the chromium update.
They were required with a previous chromium revision to be able to launch its instance
